### PR TITLE
add seqsero2 recipe

### PR DESCRIPTION
**Pull request by LSTUGA**
Bioconda requires reviews prior to merging pull-requests (PRs). To facilitate this, once your PR is passing tests and ready to be merged, please add the `please review & merge` label so other members of the bioconda community can have a look at your PR and either make suggestions or merge it. Note that if you are not already a member of the bioconda project (meaning that you can't add this label), please ping `@bioconda/core` so that your PR can be reviewed and merged (please note if you'd like to be added to the bioconda project). Please see https://github.com/bioconda/bioconda-recipes/issues/15332 for more details.

* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/contributor/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/seqsero2/build.sh [hunk 1]
```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+$PYTHON setup.py install
```
Comment (bgruening) [inline recipes/seqsero2/build.sh]: can you move this into the meta.yaml and add a `pip install ....` instead like other python only package?

File: recipes/seqsero2/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  url: https://github.com/LSTUGA/seqsero2_test/archive/v{{ version }}.tar.gz
+  sha256: '{{ sha256 }}'
+
+build:
+  number: 0
+  noarch: generic
+
+requirements:
+  host:
+    - python >=3
```
Comment (bgruening) [inline recipes/seqsero2/meta.yaml]: only python and pip are needed in the host section

File: recipes/seqsero2/build.sh (hunk 1 above)
Comment (LSTUGA) [reply recipes/seqsero2/build.sh]: Yes, of course. Thanks!

File: recipes/seqsero2/meta.yaml (hunk 2 above)
Comment (LSTUGA) [reply recipes/seqsero2/meta.yaml]: Thanks, will revise it.

File: recipes/seqsero2/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+  name: '{{ name | lower }}'
+  version: '{{ version }}'
+
+source:
+  url: https://github.com/LSTUGA/seqsero2_test/archive/v{{ version }}.tar.gz
+  sha256: '{{ sha256 }}'
+
+build:
+  number: 0
+  noarch: generic
```
Comment (dpryan79) [inline recipes/seqsero2/meta.yaml]: You'll want this to be `noarch: generic` instead, so conda will move things around appropriately for anything python version.

Comment (LSTUGA) [reply recipes/seqsero2/meta.yaml]: Okay, thanks!

File: recipes/seqsero2/meta.yaml (hunk 2 above)
Comment (LSTUGA) [reply recipes/seqsero2/meta.yaml]: @bgruening I tried to add 'salmid=0.1.23' to my dependencies, which has already been merged in bioconda recipes (https://github.com/bioconda/bioconda-recipes/pull/11868). However, the conda_build command failed to add 'salmid=0.1.23' (see the error message below). Do you know why?

03:33:19 BIOCONDA INFO (OUT) conda_build.exceptions.DependencyNeedsBuildingError: Unsatisfiable dependencies for platform linux-64: {'salmid=0.1.23'}
03:33:21 BIOCONDA ERROR COMMAND FAILED (exited with 1): docker run -t --net host --rm -v /tmp/tmpt8ny6mkg/build_script.bash:/opt/build_script.bash -v /home/circleci/project/miniconda/conda-bld/:/opt/host-conda-bld -v /home/circleci/project/recipes/seqsero2:/opt/recipe -e LANG=en_US.UTF-8 -e HOST_USER_ID=1001 bioconda/bioconda-utils-build-env:latest /bin/bash /opt/build_script.bash

03:33:21 BIOCONDA ERROR BUILD FAILED recipes/seqsero2
03:33:21 BIOCONDA INFO (COMMAND) conda build purge
03:33:22 BIOCONDA ERROR BUILD SUMMARY: of 1 recipes, 1 failed and 0 were skipped. Details of recipes and environments follow.
03:33:22 BIOCONDA ERROR BUILD SUMMARY: FAILED recipe recipes/seqsero2
Exited with code 1

Comment (dpryan79) [reply recipes/seqsero2/meta.yaml]: No one has added salmid to bioconda.

Comment (LSTUGA) [reply recipes/seqsero2/meta.yaml]: I found a salmid recipe as below. Is possible to add it to bioconda?
http://bioconda.github.io/recipes/salmid/README.html

Comment (dpryan79) [reply recipes/seqsero2/meta.yaml]: That's odd, I'll try to get that built today.

Comment (dpryan79) [reply recipes/seqsero2/meta.yaml]: The package should be available in ~25 minutes. I'll try to hit rerun on circleci once it appears.

Comment (dpryan79) [reply recipes/seqsero2/meta.yaml]: It's there now, so this should pass testing.

**Comment by dpryan79** [comment]
@bioconda-bot merge

**Pull request merged**